package.json

```diff
@@ -45,9 +46,13 @@
     "jsdom": "^20.0.0",
     "jspdf": "^2.5.1",
     "jszip": "^3.2.2",
+    "less": "^4.1.3",
+    "less-loader": "^11.1.3",
     "loaders.css": "^0.1.2",
     "lodash": "^4.17.20",
     "next": "^12.2.5",
+    "next-auth": "^4.22.3",
```

Comment:
As per https://stackoverflow.com/a/74340164, the current version of `next-auth` won't work with `next^12.2.5`. Please ensure the dependencies are met.